### DISCUSSION TITLE
What is the advantage of .Ksplat?

### DISCUSSION THREAD

**Discussion by JonaTh4nX** [mkkellogg/GaussianSplats3D#193] [Q&A]
From what I understand you create own custom .ksplat format.
What is the difference between a .ksplat file and a .splat file type? Does it have a specific optimization? I've seen that its size is generally midway between that of a .ply and a .splat.

**Comment by gotoeasy** [comment]
The ability to adjust the Minimum alpha to 100 or higher can be particularly beneficial for use on low-performance smartphones with small screens. This adjustment reduces the file size while ensuring that the content remains visually appealing on the device. The author's approach in implementing this feature is highly commendable.
